src/main/java/carRacing/domain/Car.java

```diff
@@ -0,0 +1,37 @@
+package carRacing.domain;
+
+import java.util.Random;
+
+public class Car {
+    private int score;
+
+    public Car(){
+        this.score = 0;
+    }
+
+    public Car(Car car){
+        this.score = car.score;
```

Comment:
```suggestion
        this(car.score);
```

생성자가 여러개 존재하는 경우 부 생성자가 주 생성자를 호출하여 초기화 하는것이 좋습니다.
이렇게 하면 중복이 줄어들고 간결해져서 유지보수성이 좋아집니다.

Reply:
정말 좋은 피드백 감사합니다ㅠㅠ 잊고 있던 중요한 개념을 다시 배운 기분이에요